views/accessibility-statement.html

```diff
@@ -96,7 +98,7 @@ <h2 class="govuk-heading-m" id="tested">
 
             <p class="govuk-body">We regularly carry out internal tests for compliance with the Web Content Accessibility Guidelines V2.1 level A and level AA using a variety of manual and automated testing tools.</p>
 
-            <p class="govuk-body">This service was audited on 6 July 2022. The audit was carried out by <a class="govuk-link" href="https://digitalaccessibilitycentre.org/">the Digital Accessibility Centre</a>. The complete end to end user journey was audited. An audit has not yet been carried out on ‘File an overseas entity update statement’.</p>
+            <p class="govuk-body">This service was audited on 6 July 2022. The audit was carried out by <a class="govuk-link" href="https://digitalaccessibilitycentre.org/">the Digital Accessibility Centre</a>. The complete end to end user journey was audited. An audit has not yet been carried out on ‘File an overseas entity update statement’ or ’Remove an overseas entity from the register’.</p>
```

Comment:
This is *very* picky - but:
 ’Remove an overseas entity from the register’  
that seems to start with a closing quotation mark rather than an opening one or is that just how it displays to me?

Reply:
No, good to be picky. You're right. Will correct this now.

Reply:
Done.

Reply:
Yup - happy with that!